### DISCUSSION TITLE
Send trivy-operator scans to Postee without Aqua console?

### DISCUSSION THREAD

**Discussion by kuwaharg** [aquasecurity/postee#417] [Q&A] [answered]
How do you send trivy-operator scans to Postee without an Aqua console?

We have trivy-operator installed in our cluster via helm. We would like to send messages to our teams channel when a critical vulnerability  is detected.  Most of the directions I can find show using an Aqua console for configuration.  We do not have an Aqua console for our setup. 

Is there another way to achieve this?

Thanks! --Gretchen

**Comment by simar7** [comment, marked as answer]
hi @kuwaharg - thanks for your question. You are right as today there isn't a way to send scan results from trivy-operator to Postee. We have thought of this and will get back to you on it. There is an open issue that you can track here https://github.com/aquasecurity/trivy-operator/issues/70

**Reply by kuwaharg** [reply]
Awesome!  I'll keep an eye on that issue.

Thanks for the quick response! --Gretchen

**Comment by simar7** [comment]
hi @kuwaharg - https://github.com/aquasecurity/trivy-operator/pull/476 was merged and will be available to use as a feature in the next release of Trivy operator. I've also written a brief guide here to help: https://github.com/aquasecurity/postee/pull/460 - when this is merged, it will be in the next released docs for Postee. Hope that helps.
